Choco/vmagent/tools/chocolateyinstall.ps1

```diff
@@ -4,7 +4,7 @@
 $packageName = $env:ChocolateyPackageName
 $toolsDir    = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
-$vmversion = "1.93.3"
+$vmversion = "1.93.4"

```

Comment:
We should always have versions in their version file, otherwise we run into out-of-sync problems.

Reply:
https://github.com/symflower/vmutils-bin/issues/3